### PR TITLE
PYTHON-434 - py2.6 compatibility for benchmark scripts

### PR DESCRIPTION
**Pull request by aholmberg**

### CODE REVIEW

File: cassandra/io/asyncorereactor.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def handle_read(self):
     def notify_loop(self):
         if not self._notified:
             self._notified = True
-            os.write(self.write_fd, 'x')
+            os.write(self.write_fd, b'x')
```
Comment (aboudreault) [inline cassandra/io/asyncorereactor.py]: I suppose this is more a py3 fix rather than  a 2.6 one, right? If not, why is this required?

**Comment by aboudreault** [comment]
+1

File: cassandra/io/asyncorereactor.py (hunk 1 above)
Comment (aholmberg) [inline cassandra/io/asyncorereactor.py]: You are correct. I noticed the issue as I was testing these changes across all versions.
